ParisBNF/abb/BNFabb122.xml

```diff
@@ -0,0 +1,369 @@
+                      </origin>
+                      <provenance>One of the owners was <persName ref="PRS13770EntonyosWaldaAbadia">ʾƎnṭonyos Walda ʾAbdiyu</persName>.
+                      The manuscript was acquired by
+                        <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
+                        sometime between <date>1840</date> <date>1848</date>.</provenance>
+                <acquisition>The manuscript entered the <placeName ref="INS0303BNF">Bibliothèque
+                  nationale de France</placeName> in
+                  <date>1902</date>.
+                  </acquisition>
+               </history>
+                  <additional>
+                     <adminInfo>
+                        <recordHist>
+                           <source>
+                              <listBibl type="catalogue">
+                                   <bibl>
+                                      <ptr target="bm:ContiRossini1914Notice"/>
+                                      <citedRange unit="page">62-63</citedRange>
+                                      <citedRange unit="number">35</citedRange>
+                                      </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Chaine1912Catalogue"/>
+                                      <citedRange unit="page">78</citedRange>
+                                      <citedRange unit="number">122</citedRange>
+                                   </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Abbadie1859Cat"/>
+                                      <citedRange unit="page">139-141</citedRange>
+                                      <citedRange unit="number">122</citedRange>
+                                   </bibl>
+                                    </listBibl>
+
+                           </source>
+                        </recordHist>
+                        </adminInfo>
+                     </additional>
+                  </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+           <listPerson>
+               <person>
+                 <persName ref="PRS1071dAbbadi" role="owner"/>
+              </person>
+              <person>
+                 <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                 </person>
+                   <person>
+                 <persName ref="PRS13770EntonyosWaldaAbadia" role="owner"/>
```

Comment:
again PRS1071dAbbadi?

Reply:
Sorry, I do not understand. Why again?

Reply:
https://github.com/BetaMasaheft/Persons/pull/534#pullrequestreview-904178717

Reply:
If you mean that Entonyos can be Antoin, yes, it is possible, but not proved yet, I made a remark in the PRS13770EntonyosWaldaAbadia, should I say smth in Abb 122?